plugins/ExamplePlugin/RecordBuilders/ExampleMetric2.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\ExamplePlugin\RecordBuilders;
+
+use Piwik\ArchiveProcessor;
+use Piwik\ArchiveProcessor\Record;
+use Piwik\ArchiveProcessor\RecordBuilder;
+use Piwik\Option;
+
+/**
+ * The RecordBuilder class processes raw data into ready ro read reports.
+ * It must implement two methods, one for aggregating daily reports
+ * aggregate() and another returning information about the record.
+ *
+ * Plugins can have more than one RecordBuilder, and should try to divide them
+ * up into the smallest number they can, while still performing as few total log aggregation
+ * queries as possible (this results in improved performance overall).
+ *
+ * For more detailed information about RecordBuilders please visit Matomo developer guide
+ * http://developer.piwik.org/api-reference/Piwik/ArchiveProcessor/RecordBuilder
+ */
+class ExampleMetric2 extends RecordBuilder
+{
+    const EXAMPLEPLUGIN_CONST_METRIC_NAME = 'ExamplePlugin_example_metric2';
+
+    /**
+     * This method should return the list of records this RecordBuilder creates. This example
+     * archives one metric, so we return some information about them.
+     */
+    public function getRecordMetadata(ArchiveProcessor $archiveProcessor): array
+    {
+        return [
+            Record::make(Record::TYPE_NUMERIC, self::EXAMPLEPLUGIN_CONST_METRIC_NAME),
```

Comment:
I know it wasn't the case before. But it could maybe make more sense to have one example for a numeric record and another for a blob record. Currently we only have two numeric records...